### PR TITLE
Add throw and drop

### PR DESCRIPTION
**Pull request by Nielsvdvoort2002**
Ball drop resembles dropping a ball with no initial velocity straight down, and bouncing it on the floor.
Ball throw resembles throwing a ball with some initial velocity at an angle to see where it lands.

### CODE REVIEW

**Comment by Jaswar** [comment]
Can you add .gitkeep to the data folders you created? This is just to keep the structure

File: generators.py [hunk 1]
```diff
@@ -187,6 +200,83 @@ def generate_pendulum_intensity_sequence(args):
...
+            position_x += velocity_x * (args.dt / args.ode_steps)
+            position_y += velocity_y * (args.dt / args.ode_steps)
+
+            if (position_y <= args.r):
+                velocity_x = 0
+                velocity_y = 0
+                position_y = args.r
+            else:
+                velocity_y -= args.g * (args.dt / args.ode_steps)
+    return sequence, positions_x, positions_y
```
Comment (Jaswar) [inline generators.py]: We should always return position and velocity. Consider making position (and velocity) a list of tuples (or a 2D list; an list of tuples might not work with numpy).

Comment (Nielsvdvoort2002) [reply generators.py]: Ah good idea, will do!

Comment (Nielsvdvoort2002) [reply generators.py]: I now return the zips of the two. So a list of tuples.

**Pull request merged**